lib/engine/game/g_1893.rb

```diff
@@ -104,6 +121,24 @@ def remove_tiles(tiles)
         end
       end
 
+      def init_corporations(_stock_market)
+        corporations = CORPORATIONS.dup
+
+        corporations.map! do |corporation|
+          Engine::G1893::Corporation.new(
```

Comment:
delete this and just set the properties appropriately

Reply:
Moved most of what was done in constructor to the setup class of 1893.
Have removed this part.